### DISCUSSION TITLE
How to run Prisma Client in Qwik-City

### DISCUSSION THREAD

**Discussion by y471n** [prisma/prisma#18177] [Q&A]
Is there any best practice or tutorial to follow to run Prisma in [QwikJS](https://qwik.builder.io/)?

**Comment by nurul3101** [comment]
Hello @y471n 👋 

From the documentation, it looks like QwikJS is a frontend framework similar to React.js.
Prisma cannot be run in browser. Are you planning to access database from front end?

**Reply by y471n** [reply]
Hi @nurul3101 
My bad, I meant Qwik City: https://qwik.builder.io/qwikcity/endpoints/
Qwik City -> QwikJS  
NextJS    -> ReactJS

But I found a way to get this running via the link I mentioned in the endpoint. The only part I'm unable to figure out is the ideal way to use Prisma, to not instantiate it with every request. 

Prisma has mentioned a small snippet for NextJS. I was wondering if we can have something similar to https://www.prisma.io/docs/guides/database/troubleshooting-orm/help-articles/nextjs-prisma-client-dev-practices#solution experience for Qwik-City as well.

**Reply by nurul3101** [reply]
I haven't tried out qwikcity, but I assume something similar could be done for qwikcity as well.
